packages/subgraph/schema.graphql

```diff
@@ -0,0 +1,8 @@
+type PoolMetadataUpdated @entity(immutable: true) {
```

Comment:
I believe we need another entity `Pool` that registers the latest `metadataCID` - that's the one apps will query.

Reply:
WDYT we add 

`metadataUpdates: [PoolMetadataUpdated!] @derivedFrom(field: "poolId") `

to Pool entity?

Reply:
`#question` @mendesfabio can you query subgraphs' gql API using params like `first: number, last: number` etc?

Reply:
> `#question` @mendesfabio can you query subgraphs' gql API using params like `first: number, last: number` etc?

@ribeirojose yea they provide a few filters: https://thegraph.com/docs/en/querying/graphql-api/#queries